ferrocene/doc/internal-procedures/src/testing-other-targets.rst

```diff
@@ -0,0 +1,218 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Testing Other Targets 
+=====================
+
+It's often possible to test targets other than your host tuple. Windows and macOS hosts can target
+Linux using `Lima <https://lima-vm.io/>`_ (Mac/Linux) or
+`WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_ (Windows). Lima can be used to test
+other architectures such as :ref:`x86_64-unknown-linux-gnu`, :target:`aarch64-unknown-linux-gnu`, or
+:target:`riscv64gc-unknown-linux-gnu`, while WSL2 can do similar with `QEMU <https://www.qemu.org/>`_.
```

Comment:
How about breaking this up like

```suggestion
It's often possible to test targets other than your host tuple. Windows and macOS hosts can target
Linux using `Lima <https://lima-vm.io/>`_ (Mac/Linux) or
`WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_ (Windows). 

- Lima can be used to test
other architectures such as :ref:`x86_64-unknown-linux-gnu`, :target:`aarch64-unknown-linux-gnu`, or
:target:`riscv64gc-unknown-linux-gnu`,
- WSL2 can do similar with `QEMU <https://www.qemu.org/>`_.
```

and I'd probably go further and specificy the WSL2 targets so they don't have to read the lima bullet point?

Reply:
I'm just gonna yank the last bit I think. It muddles the messaging.

Reply:
> Windows and macOS hosts can test Linux hosts using `Lima <https://lima-vm.io/>`_ (macOS/Linux) or
> `WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_ (Windows). While Lima supports other
> instruction set architectures, WSL2 users must rely on QEMU.